rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2683,6 +2684,18 @@ def test_upgrade_db_45_to_46(user_data_dir: 'Path', messages_aggregator):
             history_event_bindings,
         ).fetchone()[0] == 1
 
+        assert cursor.execute(
+            'SELECT entry_type, event_identifier, sequence_index, '
+            'timestamp, location, location_label, asset, amount, usd_value, notes, '
+            'type, subtype, extra_data FROM history_events WHERE entry_type=6',
+        ).fetchall() == [
+            (6, '20522c693bcda4ef646682c6a58bb0349b01f4d7b9168a62ce94b2c8dd1fe639', 0, 1644244023000, 'B', None, 'ATOM', '3.40000000', '0', 'Withdraw 3.40000000 ATOM from Kraken', 'withdrawal', 'remove asset', '{"reference": "KRAKEN-XXX"}'),  # noqa: E501
+            (6, '20522c693bcda4ef646682c6a58bb0349b01f4d7b9168a62ce94b2c8dd1fe639', 1, 1644244023000, 'B', None, 'ATOM', '0.10000000', '0', 'Pay 0.10000000 ATOM as Kraken withdrawal fee', 'withdrawal', 'fee', None),  # noqa: E501
+            (6, '1cf3fc675d4835efe532d18730db35163b07fb5f44956b888d7aaf852b04fd1c', 0, 1727542684000, 'G', None, 'ETH', '0.07126231', '0', 'Deposit 0.07126231 ETH to Coinbase', 'deposit', 'deposit asset', '{"address": "0xc37b40ABdB939635068d3c5f13E7faF686F03B65", "transaction_id": "0xc8aa60c9f8a93692b66fbc3f57c64f2b1c4afd92f370e12dc7bb23acfa303dbb"}'),  # noqa: E501
+        ]
+        # check that asset movements table got removed. Uncommented when we address the tables removal  # noqa: E501
```

Comment:
```suggestion
        # TODO: Remove this when table is removed
        assert cursor.execute('SELECT COUNT(*) FROM asset_movements').fetchone()[0] == 0
        # check that asset movements table got removed. TODO: Uncomment when we address the tables removal  # noqa: E501
```

Reply:
you did not do this. You just added the comment to remove the assertion, without the assertion itself.

`assert cursor.execute('SELECT COUNT(*) FROM asset_movements').fetchone()[0] == 0`